Gemfile.lock

```diff
@@ -114,6 +114,9 @@ GEM
       activesupport (>= 4.1.0)
     haml (4.0.7)
       tilt
+    httparty (0.13.7)
+      json (~> 1.8)
+      multi_xml (>= 0.5.2)
```

Comment:
While at it, you could as well squash this commit into the other one that touches `Gemfile`.